contact.html

```diff
@@ -41,7 +41,7 @@ <h1 class="">ProgrammerGaurav</h1>
 						<a title="SKills Navigation" class="nav-link" href="/#skills">Skills</a>
 					</li>
 					<li class="nav-item active">
-						<a title="Contact Navigation" class="nav-link" href="">Contact</a>
```

Comment:
the page itself is on contact.html so no need  redirect again on contact.html

Reply:
Sorry, you're right, I've edited the wrong file. You have an issue with the 'Contact' link on your nav bar, when you click it doesn't direct you to the contact page. I've fixed this now.